app/main.py

```diff
@@ -1,2 +1,44 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.len_dict = 0
+        self.dict_load = 2 / 3
+        self.list_dict = [None] * self.capacity
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        if self.len_dict > self.capacity * self.dict_load:
+            self.capacity *= 2
+            new_list_dict = self.list_dict[:]
+            self.list_dict = [None] * self.capacity
+            self.len_dict = 0
+            for i in new_list_dict:
+                if i is not None:
+                    self.__setitem__(i[0], i[1])
```

Comment:
It's better to separate this part to different function which will be responsible for resizing dict.